Emrald-UI/src/components/forms/StateForm/StateForm.tsx

```diff
@@ -64,6 +108,43 @@ const StateForm: React.FC<StateFormProps> = ({ stateData }) => {
           desc={desc}
           setDesc={setDesc}
         />
+        {diagramType === 'dtSingle' && (
+          <FormControl
+            component="fieldset"
+            sx={{
+              minWidth: 120,
+              width: '100%',
+              border: 1,
+              p: 1,
+              borderRadius: 1,
+            }}
+          >
+            <FormLabel component="legend">
+              Default Logic Tree Evaluation Value
+            </FormLabel>
+            <RadioGroup
```

Comment:
From what I remember seeing with these buttons the radio was a little close to your border line. If you could add a little bit of margin to the left that would help.

```suggestion
            <RadioGroup sx{{ml: 2}}
```